extract_unique_sensors_01.py

```diff
@@ -62,17 +62,28 @@ def extract_all_sensors(sensors_data, sensor_name):
     return candidate_lines
 
 # Loop over candidate_lines and extract sensor names
-def splitting_list(candidate_sensors, sensor_name):
-    """Splits the candidate_lines by dot and append that to sensors_list. It also removes the 
+def splint_extracted_sensors(candidate_sensors, sensor_name):
+    """Splits the candidate_lines by dot and append results to the sensors_list. It also removes the 
```

Comment:
What `candidate_lines` are being split?
Update docstring please!

Reply:
I updated the docstring.